src/components/views/passwordReset/NewPassword.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { applyReset } from '@Redux/actions/passwordResetActions';
+import InputField from '@Common/form/InputField';
+
+const ResetPassword = ({
+  loading, success, message, resetPassword, match: { params },
+}) => {
+  const [formData, setFormData] = useState({
+    password: '',
+    confirmPassword: '',
+  });
+
+  const { password, confirmPassword } = formData;
+  const onChange = event => setFormData({ ...formData, [event.target.name]: event.target.value });
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    resetPassword(password, confirmPassword, params.token);
+  };
+
+  const error = !success && message;
+  return (
+    <div className="passwd-container">
+      <Link to="/">
+        <span className="brand-logo logo">
+          <img src="https://res.cloudinary.com/mazus/image/upload/v1567523357/logo_white_background_small.png" alt="logo" />
```

Comment:
What did you say is the reason the link to this logo can't be an `env` variable?